tests/admin/controller/test_controller.py

```diff
@@ -862,39 +858,39 @@ def test_diagnostics(self):
         eq_(set(response.keys()), set(["coverage_provider", "monitor", "script", "other"]))
 
         cp_service = response["coverage_provider"]
-        cp_name, cp_collection = cp_service.items()[0]
+        [(cp_name, cp_collection)] = cp_service.items()
```

Comment:
In Python 3, `dict.items()` returns an instance of `dict_items`, which can't be indexed numerically. Fortunately there's only one key-value pair in these dicts, so unrolling them is easy.